Unity-Technologies/ScriptableRenderPipeline
# Fix for case 1168215

Basically made sure all normalization needs to happen after clipping when in the add pass.

### Purpose of this PR
Issue that appears with normalization in multi-pass in terrain shader on Universal.  This shows up when there are more than 4 layers in the terrain splatmaps.

### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?
Manually tested on the sample test case.  Manipulated the test case so that the layers are color-coded in order to have a little more clarity to verify that indeed things were showing up correctly.

Result before the fix with 6 layers  --
![MultiPassBefore](https://user-images.githubusercontent.com/44006722/61092000-43baa380-a3f9-11e9-9e50-7f0c5e353b7c.JPG)

Corrected end result after fix --
![MultiPassTest](https://user-images.githubusercontent.com/44006722/61091632-c04c8280-a3f7-11e9-8087-c4a94dc3e4e9.JPG)

Launch Katana (Link on master here - update for your branch):
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=master&automation-tools_branch=master&unity_branch=trunk&sort=1-asc
